impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -231,7 +231,7 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
         }
 
         if (Strings.hasText(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME))) {
-            Set<String> scopes = new HashSet<>(Arrays.asList(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME).split(" ")));
+            Set<String> scopes = new HashSet<>(Arrays.asList(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME).split("[\\s,]+")));
```

Comment:
@kenyattaclark thanks! Request you to sign the CLA (See https://github.com/okta/okta-sdk-java/blob/master/CONTRIBUTING.md).

Reply:
Verified the fix. It looks good.

Reply:
CLA signed and sent.

Reply:
Received it. Thanks!